.github/workflows/hail_update.yaml

```diff
@@ -17,8 +20,8 @@ jobs:
       DOCKER_BUILDKIT: 1
       BUILDKIT_PROGRESS: plain
       CLOUDSDK_CORE_DISABLE_PROMPTS: 1
-      DRIVER_IMAGE: australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:${{ github.sha }}-hail-${{ github.event.inputs.hail_version }}
-      SERVER_IMAGE: australia-southeast1-docker.pkg.dev/analysis-runner/images/server:${{ github.sha }}-hail-${{ github.event.inputs.hail_version }}
+      DRIVER_IMAGE: australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:${{ github.sha }}-hail-${{ github.event.inputs.hail_version }}-${{ github.event.inputs.hail_sha }}
```

Comment:
Could we make these in the version the short sha? Docker tags have a max 128 length sha, and 2 shas + other stuff is sitting around 95.

Reply:
Hmm, then we'd have to extract the short SHA from the full SHA using an additional step that extracts those bytes. Certainly doable, but alternatively I could also strip the `hail_version`, as that's redundant given `hail_sha`.